apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/BlockchainProcessorImpl.java

```diff
@@ -1246,21 +1250,30 @@ private void scan(int height, boolean validate, boolean shutdown) {
                 }
                 return;
             }
+            int shardInitialHeight = blockchain.getShardInitialBlock().getHeight();
```

Comment:
I suppose NPE is possible here. When blockchain.getShardInitialBlock() = NULL

Reply:
Should not be null, because at least one block exists in database